spinny_gun.py

```diff
@@ -342,10 +396,12 @@ def game_menu():
 
 
 def game_loop():
+    global pause
     """The main game loop"""
 
     """This is for the in-game background music"""
-    pygame.mixer.music.load("assets/audio/bensound-endlessmotion.mp3")
+    pygame.mixer.music.load("assets/audio/bensound-endlessmotion.wav")
+    pygame.mixer.music.set_volume(0.25)
```

Comment:
```suggestion
    pygame.mixer.music.set_volume(0.5)
```

Was this too loud on yours? It was ok on mine. I feel like 0.25 is a little low